koku/sources/sources_http_client.py

```diff
@@ -243,23 +253,48 @@ def build_source_status(self, error_obj):
         json_data = {"availability_status": status, "availability_status_error": user_facing_string}
         return json_data
 
+    def build_status_header(self):
+        """Build org-admin header for internal status delivery."""
+        try:
+            encoded_auth_header = self._identity_header.get("x-rh-identity")
+            identity = json.loads(b64decode(encoded_auth_header))
+            account = identity["identity"]["account_number"]
+
+            identity_header = {
+                "identity": {
+                    "account_number": account,
+                    "type": "User",
+                    "user": {"username": "cost-mgmt", "email": "cost-mgmt@redhat.com", "is_org_admin": True},
+                }
+            }
+            json_identity = json_dumps(identity_header)
+            cost_internal_header = b64encode(json_identity.encode("utf-8"))
+
+            return {"x-rh-identity": cost_internal_header}
+        except (binascii.Error, json.JSONDecodeError, TypeError, KeyError) as error:
+            LOG.error(f"Unable to build internal status header. Error: {str(error)}")
+
     def set_source_status(self, error_msg, cost_management_type_id=None):
         """Set the source status with error message."""
+        status_header = self.build_status_header()
+        if not status_header:
+            return False
```

Comment:
Do we not want to raise `SourcesHTTPClientError` here?

Reply:
Yeah.. I thought about that.  I didn't want it to continuously retry since this could be thought of as a "best effort" status push.

Reply:
Sounds good to me!